apps/openmw/mwmechanics/aipursue.cpp

```diff
@@ -50,8 +50,13 @@ bool AiPursue::execute (const MWWorld::Ptr& actor, CharacterController& characte
 
     //Set the target desition from the actor
     ESM::Pathgrid::Point dest = target.getRefData().getPosition().pos;
+    ESM::Position aPos = actor.getRefData().getPosition();
 
-    if (pathTo(actor, dest, duration, 100)) {
+    float dist = MWMechanics::distance(dest,aPos.pos[0],aPos.pos[1],aPos.pos[2]);
+
+    if (pathTo(actor, dest, duration, 100) &&
+        dist < MWBase::Environment::get().getWorld()->getMaxActivationDistance())  // check the true distance in case the target is far away in Z-direction
```

Comment:
We can't use activationDistance for the one check and 100 for the other check. If someone changes `iActivateDist` to < 100 then this if condition will never complete, right?

So what value should we use, anyway? 100 vs. 196 is a pretty big difference. The latter would make it much harder to escape arrest so, I'd prefer to keep the original value unless we know the larger one is actually what vanilla uses.